CONTRIBUTING.md

```diff
@@ -18,12 +24,121 @@ Possible ways to contribute:
 * CNCF SIGs & working groups (various tasks)
 * Technical content for website
 
-If you are interested in engaging in this way, we would encourage you to issue a pull request to [TOC Contributors](https://github.com/cncf/toc/blob/master/CONTRIBUTORS.md) that you desire to become a TOC Contributor. Although there is not an actual limit of having one Contributor per company, we would encourage CNCF member companies to designate an official TOC Contributor who is tasked with consulting internal experts and expressing a semi-official view on a given project. We will list current TOC Contributors on a page similar to https://www.cncf.io/people/ambassadors/.
+If you are interested in becoming a TOC member, we encourage you to create a Pull Request adding yourself to the list of [TOC Contributors](#toc-contributors). Although there is no limit of Contributors per company, we encourage CNCF member companies to designate an official TOC Contributor tasked with consulting internal experts and expressing a semi-official view on a given project.
+
+This is not only about individual contribution.  It is also about rallying help from your employer, for example, if you work for a CNCF Member company. Given the [breadth](https://landscape.cncf.io/) of projects represented in cloud native technology, it is impossible for anyone to be an expert in all technologies that we’re evaluating. We’re particularly interested in contributors who can act as a focal point for tapping relevant expertise from their organizations and colleagues in order to engage with CNCF discussions in a timely manner.
+
+The TOC already has the pattern of encouraging non-members to make non-binding votes, so no change in the TOC charter is necessary to allow contributors.
+
+TOC [meetings](https://github.com/cncf/toc#meeting-time) are only two hours a month, and are mainly taken up with project presentations. While contributors are welcome to share their views during the meeting, the biggest opportunity for comment is on the TOC [mailing list](https://github.com/cncf/toc#mailing-list), on Pull Requests representing project applications, and in voting.
+
+Just as the biggest contributors to open source projects often become maintainers, becoming an active TOC contributor is one of the best paths to distinguish oneself ahead of TOC [elections](https://github.com/cncf/toc/blob/master/process/election-schedule.md).
+
+## TOC contributors
+
+Find out more about the role of TOC contributors [here](https://github.com/cncf/toc/blob/master/CONTRIBUTING.md#toc-contributors)
+
+List below is the official list of TOC contributors, in alphabetical order:
```

Comment:
Personally I'd prefer to see the TOC Contributors list remain as a separate file - easier to read (IMO) and easier to quickly review PRs as people add themselves